File: src/classes/api/req_res/reset-password-response.ts
'use strict'

import { GkillAPIResponse } from '../gkill-api-response'

export class ResetPasswordResponse extends GkillAPIResponse {

    password_reset_path_without_host: string

    constructor() {
        super()
        this.password_reset_path_without_host = ""

    }

}

File: src/server/gkill/api/req_res/reset_password_response.go
package req_res

import "github.com/mt3hr/gkill/src/server/gkill/api/message"

type ResetPasswordResponse struct {
	Messages message.GkillMessages `json:"messages"`

	Errors message.GkillErrors `json:"errors"`

	PasswordResetPathWithoutHost string `json:"password_reset_path_without_host"`
}
